.github/workflows/smoke-test.yaml

```
@@ -6,45 +6,10 @@ on:
-
-      - name: Set output
-        id: set_output
-        run: |
-          ONLY_DOCS=$(cat result.txt)
-          echo "only_docs=$ONLY_DOCS" >> $GITHUB_OUTPUT
           
-  docker-build-and-run:
-    needs: skip_if_only_docs
-    if: needs.skip_if_only_docs.outputs.only_docs == 'false'
-    
+  docker-build-and-run:    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)